### PR TITLE
jobs: separate description from command

### PR DESCRIPTION
**Pull request by RaduBerinde**
This change adds a `Command` field to `jobspb.Payload` (and a
corresponding `command` column to `crdb_internal.jobs`). The stats job
now shows the internal command in the expanded job view. If the
command isn't present, the description is assumed to be the command.

Release note: None

Example:
image

### CODE REVIEW

File: pkg/ui/src/views/jobs/index.tsx [hunk 1]
```diff
@@ -276,7 +276,7 @@ class JobsTable extends React.Component<JobsTableProps> {
     return (
       <div>
         <h3>Command</h3>
-        <pre className="job-detail">{job.description}</pre>
+        <pre className="job-detail">{job.command ? job.command : job.description}</pre>
```
Comment (mjibson) [inline pkg/ui/src/views/jobs/index.tsx]: nit: I think this could be simplified to `job.command || job.description`

**Review by madelynnblue — APPROVED** [review]

**Review by RaduBerinde — COMMENTED** [review]
Updated to add the command column to `SHOW JOBS` as well. CC @knz 

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/35439#-:-L_ISkn22dONO4TSk61p:boeaj9h)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @vilterp)

**Comment by RaduBerinde** [comment]
Docs for SHOW JOBS will need to be updated with the `command` column (it shows the internal SQL command, unless it's the same with the description).

**Comment by knz** [comment]
The code and overall change LGTM.

I do have an issue with the word "command" though. You are introducing a new word without precedent throughout our product. Because you do that we'll have to handle a new type of question: "what's the difference between a command and a SQL statement?"

In my opinion, until jobs can be defined to do more things than SQL, we should not use a new word. The word "Statement" or "Query" would be fine.

**Comment by RaduBerinde** [comment]
I used `command` because that's what the UI shows in the expanded job view. I can change to `statement`.

**Comment by knz** [comment]
Unless there was a good reason to choose "command" originally (I don't recall such a reason, but perhaps @piyush-singh or @vilterp recall), I'd recommend changing it yes.

**Comment by RaduBerinde** [comment]
Done - renamed to "statement" everywhere. @vilterp let me know if the UI change is ok with you.

**Review by celiala — APPROVED** [review]
:lgtm:

Synced with @piyush-singh - 👍on using Statement over Command.

**Comment by RaduBerinde** [comment]
Thank you!

bors r+

**Pull request merged**